### PR TITLE
Blog post on Uber book

### PR DESCRIPTION
**Pull request by v**
Goals of this blog post

- get people excited to read the book
- share interesting snippets from the book (and have them make sense to someone that hasn't read the original text).

### CODE REVIEW

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Book Review - Super Pumped: The Battle for Uber, Mike Isaac"
+date: 2022-09-12
+published: true
+---
+
+[This book](https://www.amazon.com/Super-Pumped-Battle-Mike-Isaac-ebook/dp/B07PZ2B85Y/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1663623752&sr=8-1) was a total page turner. It has everything you'd want from a good story
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: `you'd` -> `you would`?  I'm not sure if your going for formal english, or trying to make the article have more of a conversational feel.  If you want a more conversational and relaxed feel then leave it as is.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+layout: post
+title: "Book Review - Super Pumped: The Battle for Uber, Mike Isaac"
+date: 2022-09-12
+published: true
+---
+
+[This book](https://www.amazon.com/Super-Pumped-Battle-Mike-Isaac-ebook/dp/B07PZ2B85Y/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1663623752&sr=8-1) was a total page turner. It has everything you'd want from a good story 
+
+1. famous, familiar characters like Travis Kalanick, Anthony Levandowski, Bill Gurley, Larry Page, Susan Fowler, Masayoshi San
+1. a series of incredible events - both good and bad
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: I'm not sure if you need `a` at the beginning of each bullet point.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+1. a behind-the-curtains peek into a familar product, and a hypergrowth tech company.
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: It looks like the list of compelling points was originally a sentence that got split up.  This changes the punctuation when in list form.  I'm don't think you need a comma after diverse, and plot.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+If any of this sounds like your cup of tea, pick up the book and start reading it! 
+
+Otherwise, keep on reading and I'll share what I enjoyed most about this book
+
+### My reasons for loving this book (spoilers!)
+
+**The book made me respect and understand Travis Kalanick**
+
+Before I read the book, I knew of Travis Kalanick from Uber's press stories over the last decade. I thought he was a douche-y, negligent businessperson that got lucky by being in the right place at the right time. I thought that he had no redeeming qualities and that I'd probably do his job better than he could.
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: `businessperson` -> `businessman`?

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+Travis founded two peer to peer file sharing companies before he worked for Uber - Scour and Red Swoosh. He dealt with all kinds of company crises - running out of money, lawsuits from music publishers, co-founder arguments, partnerships and negotiating acquisitions. 
+
+> At one point, when Red Swoosh was running out of money, an employee dipped into the company’s payroll tax withholdings—money a company reserves to pay the IRS the taxes it owes—to fund operations.
+
+> He was later informed by an advisor that the company might be committing tax fraud. This would stick with Kalanick for years; he felt betrayed, put in legal jeopardy by a colleague
+    
+> That cycle repeated for the next few years at Red Swoosh. Kalanick would run out of money, then secure a last-minute deal with a larger tech company and keep his business alive for another few months. He’d then find a way to parlay that deal into yet another venture investment, bailing out the company for a year or so longer.
+
+I respect people that have gone through the trenches of company building. Real experience like this hardens you.
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: I would reword this as "I respect people that have gone through the trenches of **_building a company_**.  Real **_experiences like these harden_** you."

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+published: true
+---
+
+[This book](https://www.amazon.com/Super-Pumped-Battle-Mike-Isaac-ebook/dp/B07PZ2B85Y/ref=tmm_kin_swatch_0?_encoding=UTF8&qid=1663623752&sr=8-1) was a total page turner. It has everything you'd want from a good story 
+
+1. famous, familiar characters like Travis Kalanick, Anthony Levandowski, Bill Gurley, Larry Page, Susan Fowler, Masayoshi San
+1. a series of incredible events - both good and bad
+1. a behind-the-curtains peek into a familar product, and a hypergrowth tech company.
+1. a diverse, international stage
+1. a high stakes power struggle with strong characters.
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: rather than "strong" I would say with **compelling/charismatic** characters.  I think it shows the ways in which they were strong/influential.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 7]
```diff
@@ -0,0 +1,179 @@
+---
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: Overall thoughts:
I enjoyed reading your first article!  In general I think it would help me to understand more about the intentions of your blog, and the message you want to get across for an individual article. 

A few Questions:
What type of blog is this?  
What do you want people to get out of this article?

Thoughts:
- My impression is this is a general thoughts blog/ book review blog.
- In terms of grammar I think there may be an issue with commas (I mess this up all the time when I write, so i’m not sure.)
- I left a few comments on how I would reword things.
- Everything seems factually accurate.  When I joined Lyft I remember how paranoid everyone was of Uber spies (they were fine to me… maybe a little suspicious at first.). I made a joke oce about Uber spies, and everyone was deadpan like “No really this was a problem for us over the past year.”  I remember going to meetings where Travis specifically talked about his distrust of Didi and how he DID NOT want any talent from Didi.  He talked about corporate spies from Didi a lot and how he found people on both payrolls.  I do think the book is pretty spot on from the quotes you posted.
- I think you quoted the book a lot, and I wanted to see more of your interpretations on the situation and your perspective as a tech employee.  I think this article reads like a teaser for the book (which may be your intention).   
- I’m not completely about quotes you posted in relation to your opinion of him changing.  You acknowledge that he’s more than a douchey business person, but the quotes support him being just a douchy business person.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+> Gurley’s idea in suing the former CEO was to invalidate Kalanick’s rights to those board seats entirely.
+
+> The votes [for the new CEO] kept coming in deadlocked; neither side would budge. Then, some believe, Cohler made a miscalculation. The Benchmark partner gave the table an ultimatum: If the board voted for Whitman, Benchmark would drop its lawsuit against Kalanick. It read to the room as an ultimatum. This was the price of peace. Instead of following a fair process to determine the best candidate, Benchmark was effectively holding the board hostage to approve the [replacement CEO] candidate of their choice.
+
+### Your Turn
+
+Pick up the book and [tell me](https://twitter.com/vwritescode) what you thought of it.
+
+They made [a TV series](https://www.imdb.com/title/tt11173006/) out of this book. I haven't seen it yet. I'll update this post when I do. If it's anything like the book, it'll be a treat.
```
Comment (oibe) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: What network?  who is `They`?

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown (hunk 7 above)
Comment (v) [reply _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: Follow up with ren liu articles.

Comment (v) [reply _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: There was a ton of engineering distrust with China.

They even thought that data centers in China could be attacking Uber at any time.

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown (hunk 4 above)
Comment (v) [reply _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: double edged person - charismatic but troubling

Comment (v) [reply _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: contemptible and arrogant - before

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+> The budgets for these black ops departments were obfuscated; Kalanick had purview over them. Nevertheless, Kalanick okayed budgets that spun into the tens of millions for surveillance activity, global operations, and information collection [...] Buying intelligence from third-party firms to gain an edge was normal.
+
+**Real people went through serious and tragic events as Uber grew in cities**
+
+Often, as white collar tech employees, we forget that our work affects real people. Some of the stories from Uber drivers, employees and customers are emotional and hard hitting.
+
+> As general manager of Uber’s Milan office, she worked overtime to convince Uber drivers to stay on the road, even as taxi operators would hail Ubers to their location, pull drivers out of their cars, and beat them. And one night when Lucini was returning home from work, she found a sign hung from a power line not far from her apartment. On the sign was Lucini’s home address, and a message calling her a prostitute who provided her “services” to Milan’s transportation chief."
```
Comment (v) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: city manager might as well be troops - soldiers in a war

File: _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+
+> [In Brazil,] Osvaldo Luis Modolo Filho, a fifty-two-year-old driver, was murdered by a teenage couple who hailed a ride using a fake name and chose to pay in cash. After stabbing Modolo repeatedly with a pair of blue-handled kitchen knives, the couple took off in Modolo’s black SUV, leaving him in the middle of the street.
+
+> At least sixteen drivers were murdered in Brazil before Kalanick’s product team improved identity verification and security in the app.
+
+**The power struggle between the Uber CEO and the board gives a glimpse into high stakes corporate politicking**
+
+I did not realize that the relationships between a CEO and a company's board could be so confrontational, and that people would construct convoluted schemes to oust a leader from a company.
+
+Uber's board was in a constant
```
Comment (v) [inline _posts/2022-09-19-book-review-super-pumped-battle-for-uber.markdown]: remove

**Review by v — COMMENTED** [review]
For each section of quotes, say what was interesting to me about this quote.

How did it change my perception, and how did this give me more context, or what did I find intriguing about it?
